.travis.yml

```diff
@@ -1,7 +1,13 @@
+language: ruby
+cache: bundler
+
+sudo: false
+
 rvm:
   - 1.9.3
   - 2.0.0
   - 2.1.2
+  - 2.2
```

Comment:
should probably just be:

- 2.0
- 2.1
- 2.2
- 2.3

and let rvm pick the latest, along with dropping 1.9.3 support